### PR TITLE
validate args for kubectl config view

### PR DESCRIPTION
**Pull request by Pingan2017**
**What type of PR is this?**
/kind bug
/sig cli

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes # kubernetes/kubectl#386

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/cmd/config/view.go [hunk 1]
```diff
@@ -99,7 +99,10 @@ func NewCmdConfigView(f cmdutil.Factory, streams genericclioptions.IOStreams, Co
 	return cmd
 }
 
-func (o *ViewOptions) Complete(cmd *cobra.Command) error {
+func (o *ViewOptions) Complete(cmd *cobra.Command, args []string) error {
+	if len(args) != 0 {
+		return cmdutil.UsageErrorf(cmd, "unexpected args: %v", args)
```
Comment (soltysh) [inline pkg/kubectl/cmd/config/view.go]: We should probably say use full "arguments" here, the same applies to `pkg/kubectl/cmd/apiresources/apiversions.go` and `pkg/kubectl/cmd/apiresources/apiresources.go` would you mind fixing the other two as well?

**Review by soltysh — CHANGES_REQUESTED** [review]

File: pkg/kubectl/cmd/config/view.go (hunk 1 above)
Comment (Pingan2017) [reply pkg/kubectl/cmd/config/view.go]: Thanks! 
Updated, please take a look.

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**